src/com/android/launcher3/Workspace.java

```diff
@@ -1390,6 +1390,9 @@ protected void onScrollChanged(int l, int t, int oldl, int oldt) {
     public void showPageIndicatorAtCurrentScroll() {
         if (mPageIndicator != null) {
             mPageIndicator.setScroll(getScrollX(), computeMaxScroll());
+            var isHotseatEnabled = PreferenceExtensionsKt.firstBlocking(mPreferenceManager2.isHotseatEnabled());
+            mPageIndicator.setVisibility(isHotseatEnabled ? VISIBLE : INVISIBLE);
+            mPageIndicator.setAlpha(isHotseatEnabled ? 1 : 0);
```

Comment:
No need to set alpha if it's already invisible, but why hide this too?
```suggestion
            mPageIndicator.setAlpha(isHotseatEnabled ? 1 : 0);
```

Reply:
Your right 

the pageindicator is showing at bottom, we need to hide it